python/__init__.py

```diff
@@ -199,6 +199,7 @@ class Panda:
 
   FLAG_HYUNDAI_CANFD_HDA2 = 1
   FLAG_HYUNDAI_CANFD_ALT_BUTTONS = 2
+  FLAG_HYUNDAI_CANFD_ALT_PEDALS = 4
```

Comment:
let's match what we do for the normal hyundai safety mode: https://github.com/commaai/panda/blob/1776165a3d902d6320965b6b6b1715bb9a25915b/python/__init__.py#L196